.changelog/v0.13.0/summary.md

```diff
@@ -3,5 +3,6 @@
 
```

Comment:
Can we change the summary in a release PR (which actually produces the changelog and bumps version)?